src/Features/Core/Portable/MetadataAsSource/MetadataAsSourceOptions.cs

```diff
@@ -34,4 +34,11 @@ internal readonly record struct MetadataAsSourceOptions()
     /// </summary>
     [DataMember]
     public bool NavigateToSourceLinkAndEmbeddedSources { get; init; } = true;
+
+    /// <summary>
+    /// Whether the generated metadata documents be generated to disk (<see langword="false"/>) or
+    /// if they will be generated as in memory workspace documents only (<see langword="true"/>).
+    /// </summary>
+    [DataMember]
+    public bool NavigateToVirtualFile { get; init; } = false;
```

Comment:
This does seem strange to me we're using the option like this to pass this through...

Reply:
It sure makes the unit tests nice though...